### PR TITLE
Added 15Puzzle

### PR DESCRIPTION
**Pull request by ADV1K**
Game Added. This is what the game looks like:
demo
score

#512

### CODE REVIEW

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Add a README.md file and don't clear the PR Template from next time

File: Scripts/Miscellaneous/15Puzzle/15puzzle.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    Step-4: Solve 10 and 14
+    Step-5: Solve 11, 12 and 15
+    Step-6: There is no Step-6!
+
+Reference:
+    https://anzeljg.github.io/rin2/book2/2405/docs/tkinter/
+    http://www.math.ubc.ca/~cass/courses/m308-02b/projects/grant/fifteen.html
+    https://stackoverflow.com/questions/34570344/check-if-15-puzzle-is-solvable
+
+'''
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/15Puzzle/15puzzle.py:24]: - Please remove this and create a README file according to the [README TEMPLATE](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md).

**Review by tusharnankani — CHANGES_REQUESTED** [review]
- Requesting you to follow the Code of Conduct, otherwise, the PRs could be marked invalid/spam.
- Please stick to the PR template.

**Comment by ADV1K** [comment]
added a clean PR. so im closing this one

**Pull request closed without merging**